src/test/java/catan/controllers/ctf/Scenario.java

```diff
@@ -199,51 +216,109 @@ public Scenario nextRandomDiceValues(List<Integer> nextRandomValues) {
         return this;
     }
 
+    public Scenario nextRandomDevelopmentCards(List<DevelopmentCard> nextCards) {
+        for (DevelopmentCard nextCard : nextCards) {
+            randomUtil.setNextDevelopmentCard(nextCard);
+        }
+
+        return this;
+    }
+
     public HexBuilder setHex(HexType hexType, Integer diceValue) {
         return new HexBuilder(this, hexType, diceValue);
     }
 
-    private void saveUsersResourcesValues(){
-        if(!trackResources){
+    private void saveUsersResourcesAndCardsValues() {
+        if (!trackResources && !trackCards) {
             return;
         }
 
         getGameDetails(1);
```

Comment:
what if we would have test cases with 2, 4, 5 or 6 players?
I think you shoud add iterator here and unit all one-meaning variables into massives (e.g p1Brick, p2Brick, p3Brick into usersBrickQuantity[]